parserator_web/views.py

```diff
@@ -7,18 +7,33 @@
 
 
 class Home(TemplateView):
+
     template_name = 'parserator_web/index.html'
 
 
 class AddressParse(APIView):
+
     renderer_classes = [JSONRenderer]
 
     def get(self, request):
-        # TODO: Flesh out this method to parse an address string using the
-        # parse() method and return the parsed components to the frontend.
-        return Response({})
+        input_string = request.GET.get('input', '')
+        try:
+            parsed_address = self.parse(input_string)
+        except ParseError as e:
+            raise ParseError(e)
+        [address_components, address_type] = parsed_address
```

Comment:
let's use a consistent syntax for tuple unpacking (here you use a list. on line 31, you don't wrap the var names) 

in practice, at DataMade, this would just be dealt with by black.

Reply:
Thanks for pointing this out! I've updated it.